Source/WebCore/platform/mediastream/gstreamer/GStreamerMediaStreamSource.cpp

```diff
@@ -417,6 +419,9 @@ class InternalSource final : public MediaStreamTrackPrivate::Observer,
         if (!m_parent || !m_isObserving)
             return;
 
+        if (receivedAudioSampleBeforeVideo())
```

Comment:
Was there a bug report about this? I'm not sure this patch should be needed, each src pad has its own offset and that should help with A/V synchro downstream in the pipeline.

Reply:
Hi @philn, would this help https://github.com/WebPlatformForEmbedded/WPEWebKit/pull/918#discussion_r966367309 ???